Xamarin.Forms.Core.UnitTests/MockPlatformServices.cs

```diff
@@ -100,6 +100,22 @@ public void BeginInvokeOnMainThread (Action action)
 				invokeOnMainThread (action);
 		}
 
+		public void BeginInvokeOnMainThread(Action action, Guid windowId)
```

Comment:
@wachs Are these two methods used any more? I think they might just be left over from the initial PR. Can we clean them up?